config/initializers/assets.rb

```diff
@@ -12,6 +12,7 @@
   i18n-strings.js email.css ie8.css ie9.css es5-shim.min.js html5shiv.js respond.min.js
   intl-tel-number/intlTelInput.css intl-tel-number/flags.png
   intl-tel-number/flags@2x.png
+  spinner.js
```

Comment:
Where does this `spinner.js` come from?

Reply:
It's from an earlier commit. Thanks for pointing it out!